drivers/serial/Kconfig.xlnx

```diff
@@ -9,6 +9,7 @@ config UART_XLNX_PS
 	depends on DT_HAS_XLNX_XUARTPS_ENABLED
 	select SERIAL_HAS_DRIVER
 	select SERIAL_SUPPORT_INTERRUPT
+	select PINCTRL
```

Comment:
Pinctrl is optional in this driver, as the pins can be routed in the FPGA.

Reply:
Removed